### PR TITLE
Add API description test for SurveySerializer

### PR DESCRIPTION
**Pull request by adam-thomas**
The huge pile of setup is kind of awful and the error output is totally useless, but it works.  I think.  I tried several ways to produce decent error output and each one failed, but suggestions are welcome :P

@incuna/backend @wytrych review please?

FYI @wytrych I made some minor changes to the JSON.  `help_text` is now `''` when empty rather than `null`, since the serializer gives `''`.  I also selfishly shortened one of the questions so I didn't have to break it over multiple lines in the test setup code.

### CODE REVIEW

File: surveys/tests/test_views_api.py [hunk 1]
```diff
@@ -20,5 +20,7 @@ def test_get(self):
         # that it's the same data here and hasn't been tampered with.  That way, if the
         # serializer is broken, only that test should fail, and this test will be
         # unconcerned by changes to the serializer format.
-        expected_data = SurveySerializer(instance=self.survey).data
+        expected_data = SurveySerializer(
+            instance=self.survey, context={'request': request}
```
Comment (KevinEtienne) [inline surveys/tests/test_views_api.py]: Small nitpick about the style, we use one item per line.

**Comment by KevinEtienne** [comment]
Looks good to me

File: surveys/tests/test_views_api.py (hunk 1 above)
Comment (adam-thomas) [inline surveys/tests/test_views_api.py]: oh whoops

**Comment by adam-thomas** [comment]
Thanks :)

**Pull request merged**